app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,76 @@
+<div class="border border-1 border-dark rounded p-4">
+  <div class="d-flex flex-column">
+    <div class="d-flex justify-content-between align-items-center mb-3">
+      <h4 class><%= @recipe.name%></h4>
+      <div class="public-btn">
+        <% if @recipe.public %>
+        <div class="form-check form-switch">
+          <input
+            class="form-check-input"
+            type="checkbox"
+            id="flexSwitchCheckChecked"
+            checked
+          />
+          <label class="form-check-label" for="flexSwitchCheckChecked"
+            >Public</label
+          >
+        </div>
+        <% end %> <% if !@recipe.public %>
+        <div class="form-check form-switch">
+          <input
+            class="form-check-input"
+            type="checkbox"
+            id="flexSwitchCheckChecked"
+          />
+          <label class="form-check-label" for="flexSwitchCheckChecked"
+            >Public</label
+          >
+        </div>
+        <% end %>
+      </div>
+    </div>
+    <p class="text-bold">
+      Cooking Time:
+      <span class="fw-bold"><%= @recipe.cooking_time %> min(s)</span>
+    </p>
+    <p class="text-bold">
+      Preparation Time:
+      <span class="fw-bold"><%= @recipe.preperation_time %> min(s)</span>
+    </p>
+    <div>
+      <p class="fw-bold mb-0">Steps:</p>
+      <p class="text-muted lh-base"><%= @recipe.description %></p>
+    </div>
+  </div>
+  <div class="d-flex justify-content-between">
+    <%= link_to recipe_shopping_list_index_path(@recipe), :class => "btn btn-dark bg-dark add mb-3" do %>
+    Generate shopping list <% end %> <%= link_to new_recipe_recipe_food_path(@recipe), :class => "btn
+    btn btn-outline-dark add mb-3" do %> Add Ingredients <% end %>
+  </div>
+  <hr class="my-1" />
+  <table class="table container rounded">
+    <thead>
+      <tr>
+        <th scope="col">Food</th>
+        <th scope="col">Quantity</th>
+        <th scope="col">Value</th>
+        <th scope="col"></th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @recipe.recipe_foods.each do |recipeFood|%> 
+      <tr>
+        <td><%= recipeFood.food.name%></th>
+        <td><%= recipeFood.quantity%><%= recipeFood.food.measurement_unit  %></td>
+        <td>$<%= recipeFood.quantity * recipeFood.food.price %></td>
+        <td class="d-flex gap-3 justify-content-end">
+        <% if can? :destroy, @recipe %>
+          <%=button_to "Delete", recipe_recipe_food_path(recipe_id: @recipe.id, id: recipeFood.id), method: :delete, class:"btn btn-danger"%>
+          <%=link_to "Modify", edit_recipe_recipe_food_path(recipe_id: @recipe.id, id:recipeFood.id), class:"btn btn-dark"%>
+        <%end%>
+        </td>
+      </tr>
+      <%end%>
+    </tbody>
+  </table>
+</div>
```

Comment:
- [ ] I can't see or view the implementation of the following features listed in the below screenshot because the Add recipe is not working. Please work on it.

![image](https://user-images.githubusercontent.com/55635977/163380358-3e58484e-0c7a-4476-8b00-3d7741ad68ef.png)